src/setuptools_scm/version.py

```diff
@@ -335,11 +342,24 @@ def get_no_local_node(_):
     return ""
 
 
-def postrelease_version(version):
-    if version.exact:
-        return version.format_with("{tag}")
-    else:
-        return version.format_with("{tag}.post{distance}")
+def customize_scheme(base_scheme, version, custom_scheme):
+    if custom_scheme is not None:
```

Comment:
this is the reason why scheme customization breaks next version guessing

the custom scheme discards that data/opportunity